app/views/reports/index.html.erb

```diff
@@ -15,7 +15,7 @@
 <div class="container">
   <div class="row">
     <div id="content">
-    <%= form_tag('/admin/reports/variants_by_order', method: 'get') do %>
+    <%= form_tag('/admin/reports/order_cycle_management/variants_by_order', method: 'get') do %>
```

Comment:
we should use a Rails url helper here, but we're too much in a hurry. I'll add an issue for it to do it later on.